ingestion/tests/integration/postgres/test_data_quality.py

```diff
@@ -217,65 +213,6 @@ def run_data_quality_workflow(
         metadata.delete(TestSuite, test_suite.id, recursive=True, hard_delete=True)
 
 
-def test_all_definition_exists(metadata, run_data_quality_workflow, db_service):
```

Comment:
Note that this tests coverage on test cases in integration tests. It achieves something different than the test implemented in this PR.

Reply:
Can you share more details?